src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,11 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> movie = session.createQuery("from Movie ", Movie.class);
```

Comment:
Consider renaming the variable 'movie' to 'query' or 'movieQuery' for better readability.